src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -2614,6 +2614,8 @@ namespace Akka.Dispatch.MessageQueues
     }
     public class UnboundedPriorityMessageQueue : Akka.Dispatch.MessageQueues.BlockingMessageQueue
     {
+        [System.ObsoleteAttribute("Use UnboundedPriorityMessageQueue(Func<object, int> priorityGenerator, int initia" +
```

Comment:
Marked all of the methods that don't have you explicitly set the `priorityGenerator` at construction time as obsolete.